tic_tac_toe/game.py

```diff
@@ -63,22 +86,53 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
-
-
+    #check for 1st diagonal
+    if _is_winning_combination(board,((0,0),(1,1),(2,2)), player):
+        return player
+    
+    #check for 2nd diagonal
+    elif _is_winning_combination(board,((0,2),(1,1),(2,0)), player):
+        return player
+    
+    #check horizontally:
```

Comment:
Your checking of horizontals and verticals here could be tidied up using for loops.